tests/plugins/test_pandalive.py

```diff
@@ -0,0 +1,24 @@
+from streamlink.plugins.pandalive import Pandalive
+from tests.plugins import PluginCanHandleUrl
+
+
+class TestPluginCanHandleUrlPandalive(PluginCanHandleUrl):
+    __plugin__ = Pandalive
+
+    should_match = [
+        "http://pandalive.co.kr/",
+        "http://pandalive.co.kr/any/path",
+        "http://www.pandalive.co.kr/",
+        "http://www.pandalive.co.kr/any/path",
+        "https://pandalive.co.kr/",
+        "https://pandalive.co.kr/any/path",
+        "https://www.pandalive.co.kr/",
+        "https://www.pandalive.co.kr/any/path",
+    ]
+
+    should_not_match = [
+        "http://example.com/",
+        "http://www.example.com/",
+        "https://example.com/",
+        "https://www.example.com/",
+    ]
```

Comment:
`should_not_match` should only contain non-matching URLs for that specific plugin. Generic negative matches are already defined globally here:
https://github.com/streamlink/streamlink/blob/099e7e4af66f9c4fc62ffe1af522d751f655938d/tests/plugins/__init__.py#L4-L8